### PR TITLE
ADD storybookOptions to the input for chromatic

### PR DESCRIPTION
**Pull request by ndelangen**
@tmeasday this is step 1 for https://github.com/chromaui/chromatic/issues/1956

Shall we schedule a moment we can tackle step 2 together maybe?

### CODE REVIEW

File: bin/tester/runtimes.js [hunk 1]
```diff
@@ -97,17 +99,26 @@ export default async function getRuntimeSpecs(url, { verbose = false } = {}) {
...
   } finally {
     // cleanup
     dom.window.close();
   }
 }
+
+const getOptions = ({ options }) => ({
+  hierarchyRootSeparator: options.hierarchyRootSeparator,
+  hierarchySeparator: options.hierarchySeparator,
+  storySort: options.storySort ? options.storySort.toString() : undefined,
```
Comment (tmeasday) [inline bin/tester/runtimes.js]: Do you think we should pass this up to the server, or should we instead just sort the stories before passing them up?

Comment (tmeasday) [reply bin/tester/runtimes.js]: (Not psyched about `eval`-ing user input in our server node process ;)

Comment (ndelangen) [reply bin/tester/runtimes.js]: Yea, I wasn't sure about it either.

Comment (tmeasday) [reply bin/tester/runtimes.js]: Ok, let's maybe just sort client side then.

**Pull request closed without merging**